recipes/signac-dashboard/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac-dashboard" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "bb4214e7f2be4185f7e0e1550f544b2b44f6ad04e426fedec754cf8fc9dae0a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/glotzer/signac-dashboard/get/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - signac
```

Comment:
The `setup.py` requires a version at least 0.8, and we do have older versions on conda-forge, so probably better to say `signac >=0.8`.

Reply:
Good point! @bdice Do we have additional explicit version dependencies? I remember that you mentioned that there are some specific versions required for the flask packages?

Reply:
Yes, you're right, we need `webassets >= 0.12.1` because there is a bug in `0.12.0`. The `webassets` package is included as a dependency of `flask-assets` but `flask-assets` only requires `webassets >= 0.11.1`, so it is probably necessary to require that dependency ourselves. Thanks for the reminder.